docker/example.env

```diff
@@ -30,6 +30,7 @@
 # GENESIS_BLOCK_URL='https://downloads.lisk.com/lisk/mainnet/genesis_block.json.tar.gz'
 # CLIENT_INSTANTIATION_MAX_WAIT_TIME=100,
 # CLIENT_INSTANTIATION_RETRY_INTERVAL=1,
+# CLIENT_ALIVE_ASSUMPTION_TIME=5 * 1000,
```

Comment:
If multiplication does not work here, better to set absolute value as the example.

Reply:
```suggestion
# CLIENT_ALIVE_ASSUMPTION_TIME=5000,
```